kcidb_io/schema/v4.py

```diff
@@ -29,582 +29,606 @@
+                        "Full shell command line used to make the build, "
+                        "including environment variables",
+                },
+                "compiler": {
+                    "type": "string",
+                    "description":
+                        "Name and version of the compiler used to "
+                        "make the build",
+                },
+                "input_files": {
+                    "type": "array",
+                    "description":
+                        "A list of build input files. E.g. configuration.",
+                    "items": {"$ref": "#/$defs/resource"},
+                },
+                "output_files": {
+                    "type": "array",
+                    "description":
+                        "A list of build output files: images, packages, etc.",
+                    "items": {"$ref": "#/$defs/resource"},
+                },
+                "config_name": {
+                    "type": "string",
+                    "description":
+                        "A name describing the build configuration options.",
+                },
+                "config_url": {
+                    "type": "string",
+                    "format": "uri",
+                    "description":
+                        "The URL of the build configuration file.",
+                },
+                "log_url": {
+                    "type": "string",
+                    "format": "uri",
+                    "description":
+                        "The URL of the plain-text build log file.",
+                },
+                "log_excerpt": {
+                    "type": "string",
+                    "maxLength": 16384,
+                    "description":
+                        "A part of the log file of the build most "
+                        "relevant to its "
+                        "outcome.",
+                    "examples": [
+                        "In file included from "
+                        "./arch/arm64/include/asm/processor.h:35:\n"
+                        "./arch/arm64/include/asm/pointer_auth.h:62:3: "
+                        "error: invalid input constraint 'rZ' in asm\n"
+                        "        __ptrauth_key_install(APIA, keys->apia);\n"
+                        "         ^\n"
+                        "./arch/arm64/include/asm/pointer_auth.h:55:2: "
+                        "note: expanded from macro '__ptrauth_key_install'\n"
+                        "        write_sysreg_s(__pki_v.lo, SYS_ ## k "
+                        "## KEYLO_EL1);     \\\n"
+                        "        ^\n"
+                        "./arch/arm64/include/asm/sysreg.h:829:37: note: "
+                        "expanded from macro 'write_sysreg_s'\n asm "
+                        "volatile(__msr_s(r, \"%x0\") : : \"rZ\" "
+                        "(__val));               \\\n"
+                        "                           ^\n"
```

Comment:
This example lost some of its formatting. It should be this instead:
```suggestion
                        "In file included from "
                        "./arch/arm64/include/asm/processor.h:35:\n"
                        "./arch/arm64/include/asm/pointer_auth.h:62:3: "
                        "error: invalid input constraint 'rZ' in asm\n"
                        "                __ptrauth_key_install(APIA, "
                        "keys->apia);\n                ^\n"
                        "./arch/arm64/include/asm/pointer_auth.h:55:2: "
                        "note: expanded from macro '__ptrauth_key_install'\n"
                        "        write_sysreg_s(__pki_v.lo, "
                        "SYS_ ## k ## KEYLO_EL1);     \\\n"
                        "        ^\n./arch/arm64/include/asm/sysreg.h:829:37:"
                        " note: expanded from macro 'write_sysreg_s'\n"
                        "        asm volatile(__msr_s(r, \"%x0\") : : \"rZ\" "
                        "(__val));               \\\n"
                        "                                           ^\n"
```